filesystems/android_sparse/README.md

```diff
@@ -0,0 +1,22 @@
+<!--
+SPDX-FileCopyrightText: 2021 Armijn Hemel <armijn+github@tjaldur.nl>
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+## android_sparse.img
```

Comment:
Naming a sample file the same as the format or any "placeholder" name (like [`kaitai_test`](https://github.com/kaitai-io/kaitai_struct_samples/pull/2/files#diff-c022cf0bd21f01258f0c168efa3f8c4f34ebfd214a8a7a3dfa028c6a79f7abb5)) does not make sense. In case we'd want to add multiple sample files for the `android_sparse` format, the file name of each sample should ideally reflect some distinctive feature of its contents. For some formats, we will want to test multiple versions of a header, so the sample name would contain the header version. If a sample contains a special chunk that others don't have, for example https://github.com/kaitai-io/kaitai_struct_formats/pull/229#issue-320080556:

> WAV with bext chunk: http://www.noiseofnorway.net/wp-content/uploads/2011/11/170139-Printer-Scanner-close-scanner-lid-Schoeps.wav

the file name could contain `-bext`. And so on. Of course that this approach can't be always used, but the idea is to give each file the best name that will be most useful for navigation.

I chose the "separate folder for each format" approach exactly to allow many sample files for one format. One sample file is often not enough to test all features, format and header variants/versions, chunk types etc., and we eventually want to cover the entire format to ensure that all parts of the .ksy specification work correctly - so we need more sample files. However, meaningful naming is necessary in this effort so that one has an idea what part of the `.ksy` the sample is meant to test just from the name.